qiskit/quantum_info/states/statevector.py

```diff
@@ -447,6 +447,23 @@ def from_label(cls, label):
                     state = state.evolve(y_mat, qargs=[qubit])
         return state
 
+    @staticmethod
+    def from_int(i, dims):
+        """Return a computational basis statevector.
+
+        Args:
+            i (int): the basis state element.
+            dims (int or list): the subsystem ``dims`` value for the
+                                returned statevector.
```

Comment:
can you elaborate more here? like if dealing with qubits, `int` is number of qubits.
And with higher dimensional systems, `list` is a list of dimensions and how many of them. 

Should this be a tuple instead of list? That's how you used it in tests.

Same comment for the density matrix method.